### DISCUSSION TITLE
How to remove 'latest posts' and 'selected publications' sections from the homepage

### DISCUSSION THREAD

**Discussion by jieliliu** [alshedivat/al-folio#1594] [Q&A] [answered]
Hi,

First of all, thanks for this beautiful template!
I have tried to modify the _config.yml to disable the 'latest_posts' in the 'collections' section. But it doesn't work, I can still see the module title on the homepage. Additionally, I did not find any places to remove or modify the 'selected publications' modules on the homepage. 

Thank you for the help!

**Comment by george-gca** [comment, marked as answer]
Not all information is found in the `_config.yml`, some specific settings are defined in the page front matter. For example, to disable latest posts and selected publications in the about page, simply set these fields to false in the [_pages/about.md](https://github.com/alshedivat/al-folio/blob/c824d128673c000ef9f0e0e77a0ae30d2c4b4568/_pages/about.md?plain=1#L17).

**Comment by instabaines** [comment]
I have the same issue and setting **latest_posts: false** in about.md did not work for me. 

Thank you for your help

**Reply by george-gca** [reply]
Your problem is with the latest posts or selected publications?

Also, if you want to completely remove some content, check our [section about it](https://github.com/alshedivat/al-folio/blob/main/CUSTOMIZE.md#removing-content).

**Reply by richik-p** [reply]
Follow up, we have these options under about.md but the option to remove latest posts is not present. Would you be knowing what the exact flag is for this? @george-gca 
```
news: false # includes a list of news items
selected_papers: false # includes a list of papers marked as "selected={true}"
social: true # includes social icons at the bottom of the page
```

**Reply by george-gca** [reply]
They are in [\_config.yml](https://github.com/alshedivat/al-folio/blob/cb0926f8371c827c6605f58ddce187289de2da1a/_config.yml#L161), but I agree they should be moved to `about.md`.

**Reply by richik-p** [reply]
Ah I see it thanks!
